src/components/ModalCard/ModalCard.tsx

```diff
@@ -0,0 +1,42 @@
+import './ModalCard.css';
+import ModalCardContent from './ModalCardContent/ModalCardContent';
+import { IModalCardProps } from '../../types/interfaces';
+import { useContext } from 'react';
+import { Context } from '../../context/context';
+
+export default function ModalCard({
+  modalActive,
+  isModalLoading,
+  deleteCardStringQuery,
+}: IModalCardProps) {
+  const { setModalActive } = useContext(Context);
+  function handlerClickModal() {
+    setModalActive(false);
+    deleteCardStringQuery();
+  }
+
+  function handlerClickModalCard(event: React.MouseEvent) {
+    event.stopPropagation();
+  }
+
+  return (
+    <div
+      className={modalActive ? 'modal modal-active' : 'modal'}
```

Comment:
чтобы не дублировать modal

```suggestion
      className={`modal ${modalActive && 'modal-active'}`}
```